Kernel/UnixTypes.h

```diff
@@ -549,7 +549,7 @@ struct alignas(4) sockaddr {
 #define S_IFSOCK 0140000
 #define UNIX_PATH_MAX 108
 
-struct alignas(4) sockaddr_un {
+struct sockaddr_un {
```

Comment:
This seems to remove something you added just in the last commit, mistaken fixup perhaps?